src/object_store/src/object/mod.rs

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![expect(
+    unexpected_cfgs,
+    reason = "feature(hdfs-backend) is banned https://github.com/risingwavelabs/risingwave/pull/7875"
+)]
+
```

Comment:
Will this cause the whole `object/mod.rs` to be excluded from the build?

Reply:
👀 what do you mean? It only configures lint here

Reply:
For some reason, vscode warns me that the code in this file is not active. That is why I am wondering whether the file is excluded.

Reply:
Seems a RA bug and already fixed.
https://github.com/rust-lang/rust-analyzer/issues/17817